components/zcash_address/src/kind/unified.rs

```diff
@@ -370,9 +606,12 @@ pub trait Encoding: private::SealedContainer {
     /// * the item list may not contain two items having the same typecode
     /// * the item list may not contain only transparent items (or no items)
     /// * the item list may not contain both P2PKH and P2SH items.
-    fn try_from_items(mut items: Vec<Self::Item>) -> Result<Self, ParseError> {
-        items.sort_unstable_by(Self::Item::encoding_order);
-        Self::try_from_items_internal(items)
+    fn try_from_items(
+        revision: Revision,
+        mut items: Vec<Item<Self::DataItem>>,
+    ) -> Result<Self, ParseError> {
+        items.sort_unstable_by(Item::encoding_order);
```

Comment:
The documentation says:
```rust
    /// Constructs a value of a unified container type from a vector
    /// of container items, sorted according to typecode as specified
    /// in ZIP 316.
```
but the "vector of container items" need not be pre-sorted. Please remove that clause from the doc.

Reply:
:heavy_check_mark:

Reply:
The one reason that it might make sense to keep this doc is that this _does_ sort, and so if you pass a different order of items, `items_as_parsed` will return the sorted order.